mediaPanel.lua

```diff
@@ -41,33 +44,33 @@ function mediaPanel:SetSelectorInfo(button, mType, filePath, fileName)
 	button:Show()
 end
 
-function mediaPanel:Display(anchor, clicked, mediaType, get, set)
-	local mediaPanel = self:GetPanel()
-	local holding = mediaPanel.holding
+function Media:Display(anchor, clicked, mediaType, get, set)
+	local Media = self:GetPanel()
```

Comment:
This looks like a find and replace error.  Should likely be:
local panel = self:GetPanel()